src/webservice.rst

```diff
@@ -27,6 +27,13 @@ for you.
 @conda-forge-admin, please lint
 -------------------------------
 
+Entering the above phrase in a PR of a feedstock will lint the PR (if it hasn't
+already).
+
+
+@conda-forge-admin, please relint
```

Comment:
`lint` and `relint` are the same.

Reply:
Thought `relint` always commented. Is that not correct?

Reply:
Both lint and relint always comments.